astlib/pprintast.ml

```diff
@@ -220,12 +221,15 @@ let is_simple_construct : construct -> bool = function
 
 let pp = fprintf
 
-type ctxt = { pipe : bool; semi : bool; ifthenelse : bool }
+type ctxt = { pipe : bool; semi : bool; ifthenelse : bool; functionrhs : bool }
```

Comment:
This is from upstream pprintast.ml -- do we have rules about how we update this file @NathanReb ?

Reply:
I think we simply keep it up to date with upstream pprintast for the version we're running.